onebusaway-android/src/main/res/values-it/strings.xml

```diff
@@ -585,18 +585,16 @@
     <string name="rt_infrastructure_problem">Segnala problema con la fermata</string>
     <string name="rt_customer_service">Contatta il Servizio clienti</string>
     <string name="rt_stop_problem">Segnala problema con la fermata</string>
-    <string name="rt_arrival_problem">Segnala problema con l\'orario d\'arrivo</string>
+    <string name="rt_arrival_problem">Segnala problema con l\'orario d\'arrivo</string>List
```

Comment:
There's a bit of invalid XML hanging out at the end of this line.

Reply:
Okay i, remove it.